portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,119 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.sps.servlets.DataServlet;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that stores and returns comments */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  //A class that holds comment information
+  public class Comments{
```

Comment:
It seems this class can be an immutable? https://www.geeksforgeeks.org/create-immutable-class-java/